### PR TITLE
WIP: RTX Lidar Support

### LINKED ISSUE #865: Add RTX Lidar sensor
Would this RTX Lidar use the "[range_sensor](https://docs.omniverse.nvidia.com/isaacsim/latest/features/sensors_simulation/ext_omni_isaac_range_sensor.html)"? 

https://github.com/isaac-sim/IsaacLab/pull/802

### PR DESCRIPTION
**Pull request by MoreTore**
# Description

Added RTX lidar support for training and inference instances. This is a WIP but as its stands its working. Works in headless mode. Let me know if im on the right/wrong path.

Fixes #865 

## Type of change

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Screenshots

lidar
image

## Checklist

- [ ] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [ ] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

File: source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/ray_caster/ray_caster.py [hunk 1]
```diff
@@ -293,3 +293,130 @@ def _invalidate_initialize_callback(self, event):
...
+            lidar_prim = lidar_prim.GetPath()
+            
+            render_prod_path = rep.create.render_product(lidar_prim, [1, 1])
+            self._render_product_paths.append(render_prod_path)
+            # create annotator node
+            rep_annotator = rep.AnnotatorRegistry.get_annotator("RtxSensorCpuIsaacCreateRTXLidarScanBuffer", do_array_copy=False)
+            rep_annotator.attach(render_prod_path)
+            # Debug draw
+            # self.writer = rep.writers.get("RtxLidarDebugDrawPointCloudBuffer")
+            # self.writer.attach(render_prod_path)
```
Comment (MoreTore) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/ray_caster/ray_caster.py:386]: Disable debug draw when in --headless mode

**Comment by MoreTore** [comment]
@jtigue-bdai If you guys want to take this over that would be good. I do not actually need rtx sensors at this point.

**Comment by jtigue-bdai** [comment]
@MoreTore thanks for letting us know. We will pick this up and add it to our list.

**Comment by MoreTore** [comment]
> Hello, Thank you very much for contributing this PR.
> I tested this PR and found that the LiDAR point cloud displayed in debug does not match the scene. 
> Is this just a display problem? 
> Or is it a problem with the LiDAR update rate or does the point cloud need to be transformed?
> Looking forward to getting a reply, thank you.
> Here is a video:
> https://github.com/user-attachments/assets/5dfa8d94-8d14-44ab-9071-ead13acc6a9a
> 
> 

I looks like it's just the refresh rate of the pointcloud and the debug draw points are persisting between updates in the same transform as they were captured.

**Comment by jtigue-bdai** [comment]
Closing this in favor of: [PR 1372](https://github.com/isaac-sim/IsaacLab/pull/1372). Thanks for the starting point @MoreTore, please see the other PR for implementation.

**Pull request closed without merging**